scripts/release/homebrew/docker/run.sh

```diff
@@ -2,6 +2,8 @@
 
 root=$(cd $(dirname $0); pwd)
 
+tdnf install -y ca-certificates
```

Comment:
We switched the docker image from `mcr.microsoft.com/mirror/docker/library/python:3.11-slim` to `mcr.microsoft.com/azurelinux/base/python:3` which doesn't intall `ca-certificates` by default.